specification/esql/query/QueryRequest.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+import { QueryContainer } from '@_types/query_dsl/abstractions'
+import { TimeZone } from '@_types/Time'
+import { Pragmas } from '@esql/_types/Pragmas'
+
+/**
+ * Executes an ESQL request
+ * @rest_spec_name esql.query
+ * @availability stack since=8.11.0 stability=experimental
+ */
+export interface Request extends RequestBase {
+  query_parameters: {
+    /**
+     * A short version of the Accept header, e.g. json, yaml.
+     */
+    format?: string
+    /**
+     * The character to use between values within a CSV row. Only valid for the csv format.
```

Comment:
```suggestion
     * The character to use between values within a CSV row. Only valid for the CSV format.
```

Reply:
Granted, that line was taken verbatim from the `esql.query.json` spec file. I'll leave to you the choice to update that file as well, if you do I'll resync the one in the spec with the one in the server repo afterward.

Reply:
I see. I think that comes from the `format=` parameter value, which is "usually" not capitalised (`?format=csv`), though it can be. But yes, it should be at least quoted there. Thanks!